tests/drivers/adc/adc_api/src/test_adc.c

```diff
@@ -366,7 +366,8 @@
 #define ADC_2ND_CHANNEL_ID  1
 
 #elif defined(CONFIG_BOARD_RPI_PICO) || \
-	defined(CONFIG_BOARD_ADAFRUIT_KB2040)
+	defined(CONFIG_BOARD_ADAFRUIT_KB2040) || \
+	defined(CONFIG_BOARD_SPARKFUN_PRO_MICRO_RP2040)
```

Comment:
I'm not an ADC contributor or anything, but this all seems weird - shouldn't we use Devicetree for this?

Reply:
@cfriedt This definitely seemed funky to me when I was asked to add it for the KB2040, but fixing it would definitely be out of scope for this PR, obviously. :shrug: